corehq/apps/export/static/export/js/models.js

```diff
@@ -302,7 +302,8 @@ hqDefine('export/js/models', [
     };
 
     ExportInstance.prototype.getFormatOptionValues = function () {
-        return _.filter(constants.EXPORT_FORMATS, function (format) {
+        const availableExportFormats = Object.assign({}, constants.EXPORT_FORMATS, constants.INTEGRATED_EXPORT_FORMATS);
```

Comment:
did you test these new changes on a regular export to ensure that google sheets doesn't show up in the dropdown? @RynhardPietersen